### ISSUE TITLE
Password shows minimum characters warning before inserting a password

### ISSUE THREAD

**Issue by JoseskVolpe** [JoseskVolpe/passwden#2]
**Description:**
When using --set argument and specifying a website without the login, after inserting a login when asked, the program doesn't wait for the user to insert a new password and already warns them about it needing at-least 6 characters

**What happens:**
> [joseskvolpe@RaposoPC ~]$ passwden -s teste
> Website: teste
> Login: teste
> Password: 
> Password or PIN must have at-least 6 characters
> Password: 

**What is expected:**
> [joseskvolpe@RaposoPC ~]$ passwden -s teste
> Website: teste
> Login: teste
> Password: 

**Version:**
0.11.0

**System information:**
```
[joseskvolpe@RaposoPC ~]$ neofetch
██████████████████  ████████   joseskvolpe@RaposoPC 
██████████████████  ████████   -------------------- 
██████████████████  ████████   OS: Manjaro Linux x86_64 
██████████████████  ████████   Host: Aspire ES1-572 V1.17 
████████            ████████   Kernel: 5.18.6-1-MANJARO 
████████  ████████  ████████   Uptime: 5 days, 11 hours, 43 mins 
████████  ████████  ████████   Packages: 1860 (pacman) 
████████  ████████  ████████   Shell: bash 5.1.16 
████████  ████████  ████████   Resolution: 1366x768, 1920x1080 
████████  ████████  ████████   DE: Plasma 5.24.5 
████████  ████████  ████████   WM: KWin 
████████  ████████  ████████   WM Theme: GlowFreeze-Red 
████████  ████████  ████████   Theme: Oxygen [Plasma], X-Vulpus-DarkRed [GTK2/3] 
████████  ████████  ████████   Icons: Obsidian-Red [Plasma], Obsidian-Red [GTK2/3] 
                               Terminal: konsole 
                               Terminal Font: Source Code Pro Light 10 
                               CPU: Intel i3-7100U (4) @ 2.400GHz 
                               GPU: Intel HD Graphics 620 
                               Memory: 9819MiB / 11838MiB 
```

**Comment by JoseskVolpe** [comment]
Resolved by commit a4cb34ad488b23911646b2100037e919abf6235d, wich adds a unlimited buffer by using getline() instead of scanf(), wich were receiving late stdin outputs as inputs

**Comment by JoseskVolpe** [comment]
Fixed on release 0.12.0
Closing
